app/helpers/evidence_helper.rb

```diff
@@ -3,9 +3,14 @@ def render_evidence_carousel_images(images)
     images.each_with_index do |image, index|
       concat(
         content_tag(:div, class: "carousel-item #{'active' if index.zero?}") do
-          image_tag(url_for(image), class: 'img-fluid img-thumbnail')
+          if image.image?
```

Comment:
Something about this does not feel right 😅. If the variable is called `image` I would expect it to be an image. Maybe we should rename it...

Reply:
jajaja , yes this file changed a lot you can see it's fixed [here](https://github.com/BC-MAR-21-ROR/proyecto-denuncia-ciudadana-proyecto-team02/pull/24/files#diff-24b2b2b46811a9063f40a52f81abad46b45fa611554cf65ed92a1c27207b33e2)